test/bdd/pkg/vc-devapi/vc_devapi_steps.go

```diff
@@ -0,0 +1,86 @@
+package vc_devapi
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/cucumber/godog"
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+
+	bddcontext "github.com/trustbloc/vcs/test/bdd/pkg/context"
+)
+
+var logger = log.New("bdd-test")
+
+type Steps struct {
+	bddContext     *bddcontext.BDDContext
+	responseStatus int
+	responseBody   []byte
+	credType       string
+}
+
+// NewSteps returns new Steps context.
+func NewSteps(ctx *bddcontext.BDDContext) *Steps {
+	return &Steps{bddContext: ctx}
+}
+
+// RegisterSteps registers VC scenario steps.
+func (s *Steps) RegisterSteps(sc *godog.ScenarioContext) {
+	sc.Step(`^I request did config for "([^"]*)" with ID "([^"]*)" and type "([^"]*)"$`, s.httpGet)
+	sc.Step(`^I receive response with status code "([^"]*)" for didconfig$`, s.checkResponseStatus)
+}
+
+func (s *Steps) checkResponseStatus(status string) error {
```

Comment:
can we perform some basic checks for didconfig from response as well? like some properties that for sure should be there...

Reply:
fixed